### DISCUSSION TITLE
How can i hide Navbar in root layout in a nested layout?

### DISCUSSION THREAD

**Discussion by grous0** [vercel/next.js#61110] [Help] [answered]
### Summary

I have my navbar component in app/layout.tsx and i have a sidebar component in app/dashboard/layout.tsx. The root layout wraps the nested layout and shows both navbar & sidebar. I want to hide navbar in app/dashboard/layout.tsx

### Additional information

```js
I using app router.

Binaries:
  Node: 21.6.0
  npm: 10.3.0
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.1.0
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: N/A
```

**Comment by makibat** [comment, marked as answer]
I suggest you take a look into [route groups](https://nextjs.org/docs/app/building-your-application/routing/route-groups). Basically your structure could look something like this:

```
├── app
│   ├── (other)
│   │   ├── ...
│   │   └── layout.tsx // Add Navbar component here
│   ├── dashboard
│   │   ├── page.tsx
│   │   └── layout.tsx
│   └── layout.tsx // Add Sidebar component in root layout
└── ...
```

Hope that helps!

**Reply by grous0** [reply]
Tried a lot of way finally worked. Thank you

**Reply by Vijaykumarparelli** [reply]
is there any other solution, Route Groups approach is working as expected but all root routes like /about, /menu all are need to do in (other) is it right way to do that?
